source/sandbox.ts

```diff
@@ -2,8 +2,8 @@ import * as Ethjs from 'ethjs';
 import * as EthjsAbi from 'ethjs-abi';
 import { Abi, AbiEvent } from 'ethjs-shared';
 
-const start_block = 2375000; // before serpent rep contract upload
-const end_block = 4090000; // after serpent rep freeze
+const start_block = 2378196; // serpent rep upload block number
+const end_block = 4086410; // serpent rep freeze block number
```

Comment:
I entered these vaguely so I didn't have to worry about boundary conditions (like the one for the bug you fixed).  Are you confident such boundary conditions don't exist and its safe to use the exact min/max?